gradleops/AndroidSeed/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,97 @@
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.VIBRATE"/>
+    
+    <!--START_PLUGINS_MANIFEST-->
+    <!--END_PLUGINS_MANIFEST-->
+    
+    <application android:label="${appName}" android:name="com.tealeaf.TeaLeafApplication"
+      android:icon="@mipmap/icon"
+      android:roundIcon="@mipmap/round_icon">
+    <!-- These are all the assumed default values; change as necessary and remove any others -->
+<meta-data android:name="gameHash" android:value="Unknown" />
+		<meta-data android:name="sdkHash" android:value="Unknown" />
+		<meta-data android:name="androidHash" android:value="Unknown" />
+		<meta-data android:name="develop" android:value="false" />
+		<meta-data android:name="appID" android:value="@string/game_name"/>
+		<meta-data android:name="fullscreen" android:value="true" />
+		<meta-data android:name="codeHost" android:value="s.wee.cat" />
+		<meta-data android:name="tcpHost" android:value="s.wee.cat" />
+		<meta-data android:name="codePort" android:value="80" />
+		<meta-data android:name="tcpPort" android:value="4747" />
+		<meta-data android:name="entryPoint" android:value="gc.native.launchClient" />
+		<meta-data android:name="useZeroConfig" android:value="false" />
+		<meta-data android:name="enableUpdater" android:value="false" />
+		<meta-data android:name="disableLogs" android:value="false" />
+		<meta-data android:name="android.max_aspect" android:value="2.1" />
+		<meta-data android:name="pushUrl" android:value="https://api.gameclosure.com/push/%s/?device=%s&amp;version=%s" />
+		<meta-data android:name="contactsUrl" android:value="https://api.gameclosure.com/users/me/contacts/?device=%s" />
+		<meta-data android:name="otherApps" android:value=""/>
+
+        <activity android:name="com.androidseed.AndroidSeedActivity"
+            android:launchMode="singleTask"
+            android:configChanges="keyboardHidden|orientation|screenSize"
+            android:label="${appName}"
+            android:screenOrientation="portrait"
+            android:windowSoftInputMode="stateUnspecified|adjustPan"
+            android:theme="@android:style/Theme.Translucent.NoTitleBar" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+            <intent-filter android:label="tealeaf_scheme">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:host="*" android:scheme="tealeaf" />
+            </intent-filter>
+            <!-- TODO remove ignoring, comment out autoVeryfy
+            remove: tools:ignore="AppLinkUrlError"
+            add: android:autoVerify="true"-->
```

Comment:
Can we remove this ?

Reply:
I will open separate issue for this to make sure it works fine

Reply:
This functionality is needed I think.
 @bijoshtj @vivekb91 can you confirm ?

Reply:
This is needed @CodeCombiner 
https://github.com/hashcube/native-android/commit/88d5d6b04f0f70c275482cb411a1f3d21231986f#diff-346839c0f3e1f08f1c7864d52e7df688

We added it so that we could launch our game for instance from a http url